Source/SelfService/Web/screens/logsScreen.tsx

```diff
@@ -35,10 +36,59 @@ import {
 import { TopNavBar } from '../components/topNavBar';
 import { HttpResponseApplication, getApplications, getApplication, HttpResponseApplications } from '../api/application';
 
-import { LogFilterMicroservice, LogFilterObject, LogFilterPanel } from '../logging/logFilter/logFilterPanel';
+import { LogFilterMicroservice, LogFilterObject, LogFilterDateRange, LogFilterPanel } from '../logging/logFilter/logFilterPanel';
 import { LogPanelRelative } from '../logging/logPanelRelative';
 import { LogPanelAbsolute } from '../logging/logPanelAbsolute';
 
+const useFilters = (initialFilters: LogFilterObject, availableMicroservices: LogFilterMicroservice[]): [LogFilterObject, React.Dispatch<React.SetStateAction<LogFilterObject>>] => {
```

Comment:
The `useFilters` hook can be moved to the logging feature folder. 
Perhaps renaming it to `useLogFilters` also makes sense?

Reply:
Done!